### PR TITLE
Update home div tag and styles

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 please review

**Comment by Mitso** [comment]
Screenshot 

screencapture-127-0-0-1-9000-1487934096662

File: freebasics/templates/base.html [hunk 1]
```diff
@@ -112,7 +112,7 @@
   <!-- footer block -->
   {% block footer %}
   <div class="footer custom-template-container__no-font">
-    <p>&copy; 2016 <span id="printsitelabel2">
+    <p>&copy; 2017 <span id="printsitelabel2">
```
Comment (smarzban) [inline freebasics/templates/base.html]: @Mitso Django allows you to get the current year in the template by using {% now "Y" %} tag, so we won't need to change it every year

**Comment by smarzban** [comment]
@Mitso left a tiny comment other wise 👍

File: freebasics/templates/base.html (hunk 1 above)
Comment (Mitso) [reply freebasics/templates/base.html]: Thanks you!

**Pull request merged**